src/Lean/Data/Lsp/LanguageFeatures.lean

```diff
@@ -252,22 +252,25 @@ inductive SemanticTokenType where
   | regexp
   | operator
   | decorator
+  -- Extensions
+  | leanSorryLike
   deriving ToJson, FromJson
 
 -- must be in the same order as the constructors
 def SemanticTokenType.names : Array String :=
   #["keyword", "variable", "property", "function", "namespace", "type", "class",
     "enum", "interface", "struct", "typeParameter", "parameter", "enumMember",
     "event", "method", "macro", "modifier", "comment", "string", "number",
-    "regexp", "operator", "decorator"]
+    "regexp", "operator", "decorator", "leanSorryLike"]
 
 def SemanticTokenType.toNat (type : SemanticTokenType) : Nat :=
   type.toCtorIdx
 
 -- sanity check
-example {v : SemanticTokenType} : open SemanticTokenType in
-    names[v.toNat]?.map (toString <| toJson ·) = some (toString <| toJson v) := by
-  cases v <;> native_decide
+-- TODO: restore after update-stage0
+--example {v : SemanticTokenType} : open SemanticTokenType in
+--    names[v.toNat]?.map (toString <| toJson ·) = some (toString <| toJson v) := by
+--  cases v <;> native_decide
```

Comment:
Since the kernel does not take options, `interpreter.prefer_native` does not help here :scream: